packages/@webex/plugin-meetings/src/common/errors/join-status-error.ts

```diff
@@ -0,0 +1,24 @@
+import {ERROR_DICTIONARY} from '../../constants';
+
+/**
+ * Extended Error object for join flow call meetinginfo api error
+ */
+export default class JoinStatusError extends Error {
```

Comment:
Why such name? it feels very generic, it doesn't really describe the error. What cases is this error planned to be used for? Is it only going to be used for webinar registration failure or some other cases too? If only for webinar registration then do we need the 2 error classes and translation between them or maybe we could just have one error that's thrown?

Reply:
I think we can use it for cover other join flow error, such as 403003 (Meeting is not allow to access since not reach JBH time), current in meeting-info-v2.ts handled several error (captcha, policy).